test/integration_test/extender_test.go

```diff
@@ -213,5 +213,10 @@ func pvcOwnershipTest(t *testing.T) {
 	}
 	require.Equal(t, true, errUnscheduledPod, "Pod should not have been schedule.")
 
+	err = volumeDriver.StartDriver(scheduledNodes[0])
+	require.NoError(t, err, "Error starting driver on scheduled Node %+v", scheduledNodes[0])
+
+	time.Sleep(3 * time.Minute)
```

Comment:
Call `volumeDriver.WaitDriverUpOnNode(stoppedNode)` instead of sleeping here
Check https://github.com/libopenstorage/stork/blob/27beff7c457e6850ae929abfecfc5979284ffb23/test/integration_test/extender_test.go#L153